pipeline/pipelines/core_generation_pipeline.py

```diff
@@ -52,34 +54,39 @@ class GoCoreProtoPipeline(code_generation_pipeline.CodeGenerationPipelineBase):
     set up.
     """
 
-    def __init__(self, **kwargs):
-        kwargs['language'] = 'go'
-        super(GoCoreProtoPipeline, self).__init__(**kwargs)
-
-    def do_build_flow(self, **kwargs):
-        flow = super(GoCoreProtoPipeline, self).do_build_flow(**kwargs)
-        flow.add(
+    def get_tasks(self, **kwargs):
+        return [
             protoc_tasks.ProtoCodeGenTask('CoreProtoGen',
                                           inject=kwargs),
             protoc_tasks.GoExtractImportBaseTask('ExtractGoPackageName',
                                                  inject=kwargs),
             protoc_tasks.GoLangUpdateImportsTask('UpdateImports',
-                                                 inject=kwargs))
-        return flow
+                                                 inject=kwargs)]
```

Comment:
One thing I'm wondering: is there a way to get rid of the `inject=kwargs` everywhere? It's not necessary to address in this PR, but something to consider.